package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "npm": "app.js",
   "scripts": {
-    "start": "npx nodemon ./src/app.js",
+    "start": "npx nodemon --experimental-worker ./src/app.js",
```

Comment:
just update nodejs)

Reply:
updated. with password hashing (no bcrypt for node > 10)